.github/workflows/node-zxc-build-release-artifact.yaml

```diff
@@ -181,19 +181,19 @@ jobs:
           java-version: ${{ inputs.java-version }}
 
       - name: Setup Gradle
-        uses: gradle/gradle-build-action@29c0906b64b8fc82467890bfb7a0a7ef34bda89e # v3.1.0
+        uses: gradle/actions/setup-gradle@dbbdc275be76ac10734476cc723d82dfe7ec6eda # v3.4.2
```

Comment:
The action ID has officially changed to `gradle/actions/setup-gradle`. 
See Note: https://github.com/gradle/gradle-build-action